### ISSUE TITLE
Readme parsing example won't compile

### ISSUE THREAD

**Issue by freddie-freeloader** [travisbrown/circe#175]
Thanks a lot for this great library! :smile:

The first example won't compile, because of ambiguous imports for `parse`:

```scala
import io.circe._, io.circe.generic.auto._, io.circe.jawn._, io.circe.syntax._
import cats.data.Xor

val json: String = ...

val doc: Json = parse(json).getOrElse(Json.empty)
```
I think `jawn.parse(json).getOrElse(Json.empty)` on the other hand is doing the job just fine or have I missed anything?

**Comment by travisbrown** [comment]
Ah, good catch! I'd be more than happy to accept a pull request to change the `io.circe.jawn._` import to `io.circe.parse._` if you want to submit one, or I can take care of it myself later today.

**Comment by freddie-freeloader** [comment]
If I import `io.circe.parse._` instead of `io.circe.jawn._` I get:
```
Error:Error:line (47)package io.circe.parse is not a value
val doc: Json = parse(json).getOrElse(Json.empty)
                ^
```
:disappointed:

**Comment by travisbrown** [comment]
This is fixed by #184, which renames the `io.circe.parse` package to `io.circe.parser` and updates the README.
